### DISCUSSION TITLE
feature: [plugin] material dev icons

### DISCUSSION THREAD

**Discussion by gipo355** [LazyVim/LazyVim#1617] [Q&A]
### Did you check the docs?

- [X] I have read all the LazyVim docs

### Is your feature request related to a problem? Please describe.

No, improving overall style.

### Describe the solution you'd like

Be able to use nvim web devicons with material icons.

### Describe alternatives you've considered

I've tried using https://github.com/DaikyXendo/nvim-material-icon but can't get it to work to override default devicons and it doesn't look really clean 
I didn't install his fork of devicons but tried to manually override the original devicons by copy pasting his whole icon list

### Additional context

one of the very few things i don't like about my nvim setup is the standard devicons for files.
They are really hard for me to see and distinguish. also pretty dark and boring.

I love the material icons (see vscode or https://github.com/DaikyXendo/nvim-material-icon) and have been trying to integrate them into nvim for a while now without success.
They also look more vibrant and "happy" being bigger size and colorful, giving me a more positive vibe all around. 
I believe they would be amazing paired with noice for example.

In any case, thank you folke for your amazing work!

**Comment by Alilacream** [comment]
Same here i really want to have these icons inetgrated into my lazyvim, sadly i also had no success
